pycalphad/io/cs_dat.py

```diff
@@ -543,6 +543,12 @@ def rename_element_charge(element, charge):
     if charge == 0:
         return f'{element}'
     elif charge > 0:
+        unwanted_characters=['(',')','[',']','{','}','+','-','.',',']  
+        element=list(element)
+        element=[i for i in element if i not in unwanted_characters]
+        element=[i for i in element if i.isdigit()==False ]
+
+        element=''.join(element)
         return f'{element}+{charge}'
     else:
         return f'{element}-{abs(charge)}'
```

Comment:
```python
def rename_element_charge(element, charge):
    element = "".join(filter(str.isalpha, element))
    if charge == 0:
        return f'{element}'
    elif charge > 0:
        return f'{element}+{charge}'
    else:
        return f'{element}-{abs(charge)}'
```

Is this representative of what you were doing here? Filtering for only alphabetical characters?

I assume we should just blanket apply that to elements, is that a fair assumption? You see a lot more DAT files than me.